lib/logstash/codecs/fluent.rb

```diff
@@ -70,31 +71,34 @@ def decode_event(data, &block)
       entries_decoder.feed_each(entries) do |entry|
         epochtime = entry[0]
         map = entry[1]
-        event = LogStash::Event.new(map.merge(
+        arr= []
```

Comment:
please try to follow the core conventions present in this file. when assigning to a variable, we use spaces before and after the equals sign, so `arr = []` instead of `arr= []`